bep028spec.md

```diff
@@ -584,33 +600,39 @@ sub-<label>/
       sub-<label>[_ses-<label>]_prov-<label>_<suffix>.<extension>
 ```
 
-Here is an example dataset tree:
+> [!TIP] Here is an example dataset tree:
+> ```
+> └─ dataset
+>   ├─ prov/
+>   │  └─ prov-dcm2niix_base.json
+>   ├─ sub-001/
+>   │  ├─ prov/
+>   │  │  └─ sub-001_prov-dcm2niix_act.json
+>   │  └─ ses-01/
+>   │     ├─ prov/
+>   │     │  └─ sub-001_ses-01_prov-dcm2niix_act.json
+>   │     └─ ...
+>   ├─ sub-002/
+>   │  ├─ prov/
+>   │  │  └─ sub-002_prov-dcm2niix_act.json
```

Comment:
in addition to the dataset level prov folder,  thought it was either prov per file, rather than prov folders everywhere. was there a specific discussion for this change that i missed?